sources/PTYTextView.m

```diff
@@ -4831,8 +4833,10 @@ - (void)doCommandBySelector:(SEL)aSelector
         // See comment in -keyDown:
         DLog(@"Rerouting doCommandBySelector from %@ to %@", self, gCurrentKeyEventTextView);
         [gCurrentKeyEventTextView doCommandBySelector:aSelector];
+        [self.delegate keyDown:_keydownEvent];
```

Comment:
Thanks for the explanation. I see why this was done this way: `doCommandBySelector:` is a more appropriate place to handle the keystroke than in `keyDown:` when possible.

I can't find a case where `-handleEvent:` returns `NO` in this code path. Can you get it to return NO? Maybe it's specific to the input method (I tried US and Japanese Katakana) or some other system configuration.